### ISSUE TITLE
Cannot create Ubuntu14.4 box

### ISSUE THREAD

**Issue by jhalterman** [u39kun/ambari-vagrant#8]
When attempting to `up.sh` an Ubuntu14.4 box:

```
Bringing machine 'u1401' up with 'virtualbox' provider...
==> u1401: Box 'ubuntu14.4' could not be found. Attempting to find and install...
    u1401: Box Provider: virtualbox
    u1401: Box Version: >= 0
==> u1401: Adding box 'ubuntu14.4' (v0) for provider: virtualbox
    u1401: Downloading: https://github.com/hnakamur/packer-templates/releases/download/v1.0.2/ubuntu-14-04-x64-virtualbox.box
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.
```

That file appears to no longer exist:

```
curl https://github.com/hnakamur/packer-templates/releases/download/v1.0.2/ubuntu-14-04-x64-virtualbox.box
{"error":"Not Found"}
```

**Comment by mikhail-antonov** [comment]
Yeah, seems like pre-built vagrant boxes aren't there anymore. What you could do (if that's still relevant) is to clone https://github.com/shiguredo/packer-templates where they've moved, and build the box for ubuntu-14-04 yourself using packer (packer.io), then modify Vagrantfile to point to the location of thss .box file on your filesystem.

**Comment by Itaypk** [comment]
Seems to be resolved now - installation of Ubuntu 14.04 worked with no problem, with the box URL taken from http://naruh.sakura.ne.jp/vagrant/ubuntu-14-04-x64-virtualbox.box
